.gitignore

```diff
@@ -5,7 +5,8 @@
 ## User settings
 xcuserdata/
 xuserdatad/
-.pbxproj
+*.pbxproj
```

Comment:
Wait hold up @HariniSundaram0, why did you add .pbxproj to your .gitignore? 😳

I am quite confident you don't want to do this, nobody is going to be able to open up your project with a clean checkout if this file is missing.

As a side note, this is why I recommend using a well-known and well-starred .gitignore file, and being very judicious with your tweaks; you can do serious damage to your project if you add the wrong files/folders here.